lld/test/COFF/arm64x-comm.s

```diff
@@ -0,0 +1,18 @@
+// REQUIRES: aarch64
+
+// Check that -aligncomm applies to both native and EC symbols.
+
+// RUN: llvm-mc -filetype=obj -triple=aarch64-windows-gnu %s -o %t-arm64.obj
+// RUN: llvm-mc -filetype=obj -triple=arm64ec-windows-gnu %s -o %t-arm64ec.obj
+// RUN: lld-link -machine:arm64x -lldmingw -dll -noentry -out:%t.dll %t-arm64.obj %t-arm64ec.obj
+// RUN: llvm-readobj --hex-dump=.test %t.dll | FileCheck %s
+// CHECK: 0x180004000 08200000 10200000 18200000 20200000
```

Comment:
I feel that this layout of the test data doesn't really showcase the alignment fully; if I read the directives correctly, we have `sym` with 2^3=8 bytes alignment, and `sym2` with 2^4=16 bytes alignment. But then the pointers end up with `.data` start at `0x2000`, `sym` at `0x2008` and `sym2` at `0x2010`. If the relative alignment of the two would be flipped, I think it would show more interesting results; `sym` at `0x2010` and `sym2` at `0x2018`, WDYT?

(I guess the testcase may be based on an existing one with the same issue though.)

Reply:
Good idea, I changed it, thanks!